packages/features/ee/workflows/lib/reminders/templates/emailReminderTemplate.ts

```diff
@@ -4,21 +4,35 @@ import { APP_NAME } from "@calcom/lib/constants";
 import { TimeFormat } from "@calcom/lib/timeFormat";
 import { WorkflowActions } from "@calcom/prisma/enums";
 
-const emailReminderTemplate = (
-  isEditingMode: boolean,
-  locale: string,
-  action?: WorkflowActions,
-  timeFormat?: TimeFormat,
-  startTime?: string,
-  endTime?: string,
-  eventName?: string,
-  timeZone?: string,
-  location?: string,
-  meetingUrl?: string,
-  otherPerson?: string,
-  name?: string,
-  isBrandingDisabled?: boolean
-) => {
+const emailReminderTemplate = ({
```

Comment:
Refactored this function to take in an object as a param for readability and to align with other template functions.